internal/events/aggregator_test.go

```diff
@@ -1634,20 +1669,38 @@ func TestDispatchBroadcastQueuesLaterDispatch(t *testing.T) {
 	mim.On("FindIdentityForVerifier", ag.ctx, mock.Anything, mock.Anything).Return(org1, nil)
 
 	mdm := ag.data.(*datamocks.Manager)
-	mdm.On("GetMessageWithDataCached", ag.ctx, msg1.Header.ID, data.CRORequirePublicBlobRefs).Return(msg1, core.DataArray{}, true, nil).Once()
-	mdm.On("GetMessageWithDataCached", ag.ctx, msg2.Header.ID, data.CRORequirePublicBlobRefs).Return(msg2, core.DataArray{}, true, nil).Once()
+	data1 := core.DataArray{}
```

Comment:
Honestly the aggregator tests are a bit rusty. When I started debugging, they were following paths that I did not expect based on the test names. I suspect they've never been fully overhauled in the wake of lots of identity changes and lots of batch state changes.

I had to touch a few tests to fix them after my changes, so I also tried to make those more consistent with what I _think_ they're supposed to be testing - but just wanted to note that there are quite a few others that are stale and might really need to do extra steps to validate things that end up in `batchState`.